### DISCUSSION TITLE
p.148 octocov設定ファイルに関する記載漏れ

### DISCUSSION THREAD

**Discussion by ac0mz** [budougumi0617/go_todo_app#43] [Ideas]
p.148では `test.yml` を作成するだけでカバレッジレポートがPRにコメントされる旨の記載がありますが、実際にはプロジェクトルート配下に `.octocov.yml` を配置しなければ、octocovがカバレッジレポートの生成を実行しません。
※以下スクショは.octocov.ymlがない状態でのCI結果です。

<img width="850" alt="failed_octocov_report" src="https://user-images.githubusercontent.com/40301275/187061659-b68ba7eb-908b-4c85-925f-a1ad95f7671e.png">

サンプルリポジトリの `.octocov.yml` を配置したらカバレッジレポートがPRにコメントされるようになりました。

**Comment by budougumi0617** [comment]
ご指摘ありがとうございます！今週中に確認・正誤表の更新をさせていただきます！とりいそぎお礼まで…

**Reply by budougumi0617** [reply]
すでに他のdiscussionsでご報告済みですが、ここでもご指摘に対して正誤表を更新したことをご報告させていただきます
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.5
https://github.com/budougumi0617/go_todo_app/blob/v1.0.5/errata.md
